.sync.yml

```diff
@@ -17,6 +17,8 @@ Gemfile:
       - gem: mocha
         version: '~>0.10.5'
       - gem: puppet-blacksmith
+      - gem: mime-types
+        version: '~> 2.99'
```

Comment:
Minor nit, but I think you want to skip the space like `'~>2.99'` to match what you've put in the `Gemfile`